regions/io/fits/write.py

```diff
@@ -0,0 +1,59 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+from astropy.io import fits
+
+from ..core import to_shape_list, SkyRegion
+
+
+__all__ = [
+    'write_fits_region',
+    'fits_region_objects_to_table',
+]
+
+
+def fits_region_objects_to_table(regions):
+    """
+    Converts list of regions to FITS region table.
+
+    Parameters
+    ----------
+    regions : list
+        List of `regions.Region` objects
+
+    Returns
+    -------
+    region_string : `~astropy.table.Table`
+        FITS region table
+
+    Examples
+    --------
+    TODO
```

Comment:
Could you fill this in with a brief example?

Reply:
on it!

Reply:
@keflavich Also, I wanted to attach `region.fits` file to be downloadable for the user in the docs. How do I do that?

Reply:
hmm... good question!  I think we want to put that on data.astropy.org... ask on slack in the general forum how to do that